### ISSUE TITLE
Windows Nim first time user experience with choosenim, dlls.zip download,extract problem.

### ISSUE THREAD

**Issue by ftsf** [dom96/choosenim#207]
With choosenim presenting itself as the easy way to get Nim, it currently gives a bad first experience for Nim on Windows.

Following the instructions: 

Going to the releases page (not seeing the note about getting the zip) and getting the exe instead expecting it to be an installer.
Attempting to run the downloaded exe, nothing appears to happen.

Reread the instructions and see the note about needing the .zip. Get that.
Run runme.bat

```
$ ./runme.bat
    Prompt: Can choosenim record and send anonymised telemetry data? [y/n]
        ... Anonymous aggregate user analytics allow us to prioritise
        ... fixes and features based on how, where and when people use Nim.
        ... For more details see: https://goo.gl/NzUEPf.
<<<WAITING ON USER INPUT HERE>>>
y
    Answer: Downloading DLLs (openssl, pcre, ...)

[                                                  ] 0.06047% 4kb/s
[#############################                     ] 58.18% 3844kb/s
[##################################################] 100.0% 0kb/s
 Extracting dlls.zip
       Tip: 1 messages have been suppressed, use --verbose to show them.
     Error: Unable to extract. Error was 'Cannot create a file when that file already exists.
        ... '.
        1 file(s) copied.
            Work finished.
            Now you must ensure that the Nimble bin dir is in your PATH:
              C:\Users\user\.nimble\bin
Press any key to continue . . .
```

run `choosenim stable`

```
$ choosenim stable
      Info: C:\Users\user\.choosenim\downloads\dlls.zip already downloaded
 Extracting dlls.zip
     Error: Unable to extract. Error was 'Cannot create a file when that file already exists.
        ... '.
```

Deleting the dlls.zip and running `choosenim stable`

```
$ rm ~/.choosenim/downloads/dlls.zip

user@Jezzycomp MINGW64 ~
$ choosenim stable
Downloading DLLs (openssl, pcre, ...)

[                                                  ] 0.06047% 4kb/s
[##########################                        ] 52.67% 3480kb/s
[##################################################] 100.0% 0kb/s
 Extracting dlls.zip
     Error: Unable to extract. Error was 'Cannot create a file when that file already exists.
        ... '.
```

Seems to be stuck in a loop that I can't do anything about.
I worked around this by using scoop install method instead, but I think the choosenim windows experience could be improved to make people's first experience of nim a more pleasurable one by making the exe be an installer, or get rid of it if the zip is required and obviously a fix for this dlls bug which seems unrecoverable at the moment.

**Comment by kerskuchen** [comment]
Running 

``` 
choosenim.exe stable
```

from inside the choosenim dir in the .zip gives me 

```
C:\nim\choosenim>choosenim.exe stable
Downloading Nim 1.2.4 from nim-lang.org
[#                                                 ] 0mb 0kb/s
      Info: Binary build unavailable, building from source
[##################################################] 100.0% 0kb/s
 Extracting nim-1.2.4.tar.xz
   Building Nim 1.2.4
   Building koch
   Cleaning failed build
oserr.nim(94)            raiseOSError
Error: unhandled exception: The system cannot find the file specified.
Additional info: "Requested command not found: \'bin/nim.exe c koch\'. OS error:" [OSError]
```

As a first time Nim user I agree with @ftsf. It is currently not as simple as i.e. rustup.

**Comment by dom96** [comment]
This should be fixed by #201 (release pending)
